assets/scss/base/_helpers.scss

```diff
@@ -224,3 +225,8 @@
   }
 
 }
+
+.truncate--fixed-small {
```

Comment:
Yes, you're right. I changed it to max-width later without updating the class name. I'll do that now.